simplified-books-api/src/main/java/org/nypl/simplified/books/api/BookmarkKind.kt

```diff
@@ -40,7 +40,7 @@ sealed class BookmarkKind(
   companion object {
 
     val obsoleteBookmarkingURI =
-      "http://www.w3.org/ns/oa#bookmarking"
+      "https://www.w3.org/ns/oa#bookmarking"
```

Comment:
At first I thought having `https` here was a mistake but I see the `obsolete` prefix now, so all good. :)